boards/FSM/Src/App/App_Brake.c

```diff
@@ -55,23 +62,48 @@ struct InRangeCheck *App_Brake_GetPressureInRangeCheck(const struct Brake *const
 {
     return brake->pressure_in_range_check;
 }
-
 bool App_Brake_IsBrakeActuated(const struct Brake *brake)
 {
     return brake->is_brake_actuated();
 }
-
 bool App_Brake_IsPressureSensorOpenOrShortCircuit(const struct Brake *brake)
 {
-    return brake->is_pressure_sensor_open_or_short_circuit();
+    // TODO perhaps rethink how this is implemented
+    return brake->is_primary_pressure_sensor_open_or_short_circuit() ||
+           brake->is_secondary_pressure_sensor_open_or_short_circuit();
 }
-
 bool App_Brake_IsPressureSensorOpenCircuit(const struct Brake *brake)
 {
-    return brake->get_pressure_psi() < BRAKE_PRESSURE_OC_THRESHOLD;
+    return brake->get_primary_pressure_psi() < BRAKE_PRESSURE_OC_THRESHOLD;
 }
-
 bool App_Brake_IsPressureSensorShortCircuited(const struct Brake *brake)
 {
-    return brake->get_pressure_psi() > BRAKE_PRESSURE_SC_THRESHOLD;
+    return brake->get_primary_pressure_psi() > BRAKE_PRESSURE_SC_THRESHOLD;
```

Comment:
Because of the electronics of the sensor and FSM, its not useful to distinguish between open circuit and short circuit. As such the most logical implementation IMO is a separate function return bool for each pressure sensor. Either in-bounds or out-of-bounds

Reply:
Fixed with a2de2930a54f2fd619c4a6549df228f06ea60214